vendor/raylib/rlgl.odin

```diff
@@ -425,7 +433,8 @@ foreign lib {
 	rlDisableScissorTest     :: proc() ---                           // Disable scissor test
 	rlScissor                :: proc(x, y, width, height: c.int) --- // Scissor test
 	rlEnableWireMode         :: proc() ---                           // Enable wire mode
-	rlDisableWireMode        :: proc() ---                           // Disable wire mode
+	rlEnablePointMode		 :: proc() --- 							 // Enable point mode
```

Comment:
Please use tabs for indentation and **spaces** for alignment. There's a few cases of you using tabs to align things which looks off if you set the tab size to 8.